src/client/Microsoft.Identity.Client/Internal/Broker/BrokerInteractiveRequestComponent.cs

```diff
@@ -93,6 +92,7 @@ private async Task<MsalTokenResponse> SendAndVerifyResponseAsync()
             BrokerPayload.Add(BrokerParameter.Username, _authenticationRequestParameters.Account?.Username ?? string.Empty);
             BrokerPayload.Add(BrokerParameter.ExtraOidcScopes, BrokerParameter.OidcScopesValue);
             BrokerPayload.Add(BrokerParameter.Prompt, _interactiveParameters.Prompt.PromptValue);
+            BrokerPayload.Add(BrokerParameter.Claims, _authenticationRequestParameters.ClaimsAndClientCapabilities);
```

Comment:
@bgavrilMS 
This looks like the reason iOS didnt work E2E with claims as the claims were simply never put into the request parameters for broker. Also, we may have to split up the client capabilities and claims on the AuthenticationRequestParameters.cs object into individual parameters depending on how the broker handles these.

Reply:
Yes, good point. Our MSAL differentiates between capabilities and claims already.